test/unittests/execute_floating_point_test.cpp

```diff
@@ -1072,7 +1073,16 @@ TEST(execute_floating_point, f32_demote_f64)
 
     for (const auto& [arg, expected] : test_cases)
     {
-        EXPECT_THAT(execute(*instance, 0, {arg}), Result(expected)) << arg << " -> " << expected;
+#pragma STDC FENV_ACCESS ON
```

Comment:
Why don't we have this in the implementation?

Reply:
Because it seems it changes absolutely nothing.

Reply:
Neither GCC nor Clang support this pragma.